e2e/app/container.ts

```diff
@@ -0,0 +1,16 @@
+import {Page} from 'puppeteer';
+
+/**
+ * This is the super base class.
+ * Every element needs a Page object and a xpath for locating the element.
+ */
+export default class Container {
```

Comment:
I don't think `Container` is quite right. Aren't there leaf elements that are xpath-able that don't contain things?

Reply:
How about something like `ComponentWrapper`? That's what we're actually locating here

Reply:
Thank you. I will consider renaming in future refactor PRs.